### PR TITLE
Update providers.yml

### PR DESCRIPTION
**Pull request by gitmanju**
Adding cryptophoto

### CODE REVIEW

File: _data/providers.yml [hunk 1]
```diff
@@ -42,7 +42,15 @@ providers:
     url: https://getclef.com
     img: clef.png
     software: Yes
-
+    
+  - name: CryptoPhoto
+    url: http://cryptophoto.com
```
Comment (Carlgo11) [inline _data/providers.yml:47]: use https

File: _data/providers.yml [hunk 2]
```diff
@@ -42,7 +42,15 @@ providers:
...
+    img: crypto.png
```
Comment (Carlgo11) [inline _data/providers.yml:48]: There's no crypto.png. Please add cryptophoto.png in img/providers. The logo <b>Must</b> be 32x32px

File: _data/providers.yml [hunk 3]
```diff
@@ -42,7 +42,15 @@ providers:
...
+    hardware: Yes
+    phone: Yes
+    sms: No
```
Comment (Carlgo11) [inline _data/providers.yml:51]: if they don't support sms you don't need to add it.

Comment (gitmanju) [inline _data/providers.yml:51]: ok will remove it. thanks

**Pull request closed without merging**